tools/apis-and-packages/xdpos2.0-apis.md

```diff
@@ -0,0 +1,216 @@
+# XDPoS 2.0 API
+
+- [Get V2 Block By Number](#xdpos_getv2blockbynumber)
+- [Get V2 Block By Hash](#xdpos_getv2blockbyhash)
+- [Get Masternodes By Number](#xdpos_getmasternodesbynumber)
+- [Get Latest Pool Status](#xdpos_getlatestpoolstatus)
+
+
+Provide `curl` command as exmaple
+
```

Comment:
highlight the attribute "commited".
Sample paragraph:

XDPoS2.0 enables fast block and transaction confirmation in the order of seconds. This confirmation can be obtained from the "committed" attribute of the result of block queries. If a block's "committed" attribute is True, then this block and the transactions in it can all be confirmed. Otherwise, they are not confirmed yet.